.teamcity/Gradle_Promotion/buildTypes/MasterSanityCheck.kt

```diff
@@ -20,7 +22,7 @@ object Gradle_Promotion_MasterSanityCheck : BuildType({
     steps {
         gradle {
             tasks = "tasks -s"
-            buildFile = ""
+            useGradleWrapper = true
```

Comment:
Why do we need this? Isn't `useGradleWrapper` default to be true? If not, how did it work before?

Reply:
We are setting`useGradleWrapper = true` in our other scripts. It seems to be somewhat the default. Let me re-use the `gradleWrapper` extension function from `GradleBuildConfigurationDefaults.kt`.